src/OpenTelemetry/Metrics/Reader/PeriodicExportingMetricReaderOptions.cs

```diff
@@ -52,4 +52,9 @@ internal PeriodicExportingMetricReaderOptions(IConfiguration configuration)
     /// associated with the metric reader.
     /// </summary>
     public int? ExportTimeoutMilliseconds { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether to use threads. Enables the use of <see cref="Thread" /> when true, <see cref="Task"/> when false. The default value is true.
```

Comment:
```suggestion
    /// Gets or sets a value indicating whether to use threads. Enables the use of <see cref="Thread" /> when <see langword="true"/>; otherwise <see cref="Task"/> is used. The default value is <see langword="true"/>.
```